.github/workflows/release.yml

```
@@ -0,0 +1,113 @@
+name: Release on Merge to Master
+
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+
+env:
+  PLUGIN_FUNCTIONS_MAP: |
+    {
+      "azure-python-promote-or-quarantine": "[\"promoteOrQuarantineFunction\"]",
+      "azure-python-slack-notification": "[\"slackNotification\"]",
+      "azure-python-teams-notification": "[\"teamsNotification\"]"
+    }
+
+jobs:
+  declare-variables:
+    runs-on: ubuntu-latest
+    outputs:
+      plugin_dir: ${{ steps.path_url.outputs.plugin_dir }}
+
+    steps:
+      - name: Declare Path URL
+        id: path_url
+        run: |
+          plugin_dir=${{ github.workspace }}/post-scan-actions
+          echo "plugin_dir=$plugin_dir" >> $GITHUB_OUTPUT
+
+  extract-changed-azure-plugins:
+    runs-on: ubuntu-latest
+    outputs:
+      changed_plugins: ${{ steps.extract_changes.outputs.changed_plugins }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get changed files
+        id: changed-files
```

If `underscore` is okay to use, maybe make the name convention consistent with IDs of other steps.
```suggestion
        id: changed_files
```